context/AuthContextProvider.tsx

```diff
@@ -89,9 +93,8 @@ export const AuthContextProvider = ({
 
     try {
       const user = await account.get();
-      const userData: IUser = await getCurrentUser(user.$id);
-      updateUser(userData.id, userData.email, userData.leagues);
-      return userData;
```

Comment:
Why is the return removed?

Reply:
✅ it is back in :).  Not sure why it was removed.